integration/query_compat_test.go

```diff
@@ -76,9 +81,8 @@ func testQueryCompat(t *testing.T, testCases map[string]queryCompatTestCase) {
 
 					if targetErr != nil {
 						t.Logf("Target error: %v", targetErr)
-						targetErr = UnsetRaw(t, targetErr)
-						compatErr = UnsetRaw(t, compatErr)
-						assert.Equal(t, compatErr, targetErr)
+						AssertEqualErrorCode(t, compatErr, targetErr)
```

Comment:
We only compare error code in compat tests.

Reply:
I understand why we started doing it (@AlekSi proposed that and everyone agreed), but I have a new question now. 

If we don't check the error message of our errors in compat tests, and we don't have non-compat tests anymore, where do we check the error message of our errors?..  I guess, for query tests it's ok'ish, especially, if error messages are just constants. But in principle, it might make sense to check some error messages (e.g. if they contain variable parts). This is a side note, just wanted to share my thoughts.

Reply:
This is a valid point, I would like to know how exactly we need to be for the variable parts.

Reply:
Let's create a new issue to think about that and what to do with existing `AssertEqualAltError`

Reply:
https://github.com/FerretDB/FerretDB/issues/1587